docker/run.sh

```diff
@@ -79,21 +79,23 @@ done
 
 # Update the arguments to default values if needed.
 
-IMAGE_NAME=${IMAGE_NAME:-ros2_humble_andino}
-CONTAINER_NAME=${CONTAINER_NAME:-ros2_humble_andino_container}
+IMAGE_NAME=${IMAGE_NAME:-ros2_jazzy_andino}
+CONTAINER_NAME=${CONTAINER_NAME:-ros2_jazzy_andino_container}
+
+USER=ubuntu
 
 SSH_PATH=/home/$USER/.ssh
-WORKSPACE_SRC_CONTAINER=/home/$(whoami)/ws/src/$REPOSITORY_FOLDER_NAME
-WORKSPACE_ROOT_CONTAINER=/home/$(whoami)/ws
+WORKSPACE_SRC_CONTAINER=/home/ubuntu/ws/src/$REPOSITORY_FOLDER_NAME
+WORKSPACE_ROOT_CONTAINER=/home/ubuntu/ws
```

Comment:
Is it going to find it on any computer if we use ubuntu as user?

Reply:
It should because the same user would be used on any computer that runs the container, it's not as we did on Ubuntu22 where we were using the current user on the host machine.